src/renderer/components/welcome.tsx

```diff
@@ -234,52 +259,38 @@ export class Welcome extends React.Component<
 
   public render() {
     const { sleuth } = this.state;
-    const scrollStyle: React.CSSProperties = {
-      marginBottom: 24,
-      overflowY: 'auto',
-      minWidth: 480,
-      width: '60%',
-    };
 
     const suggestions = this.renderSuggestions();
 
     return (
-      <div
-        className="Welcome"
-        style={{ justifyContent: suggestions ? undefined : 'end' }}
-      >
+      <div className="welcome css-var-">
         <div>
-          <h1 className="Title">
-            <span className="Emoji">{sleuth}</span>
-            <span>Sleuth</span>
-          </h1>
-          <Typography.Title level={4}>
-            Drop a logs zip file or folder anywhere on this window to open it.
+          <Typography.Title level={1} className="welcome__title">
+            <span className="welcome__title-emoji">{sleuth}</span> Sleuth
           </Typography.Title>
         </div>
 
         {suggestions ? (
-          <>
-            <Typography.Title level={5}>
-              From your Downloads folder, may we suggest:
-            </Typography.Title>
-            <div style={scrollStyle}>
-              <div style={{ textAlign: 'initial' }}>{suggestions}</div>
+          <div className="welcome__suggestion-container">
+            <div className="welcome__suggestion-drag-and-drop-reminder">
+              (or drag and drop logs)
```

Comment:
fwiw, this is a bit confusing since there's no other clear call to action

Maybe the Downloads tab could say `Open from <downloads-path>`

Reply:
[b3b3258](https://github.com/tinyspeck/sleuth/pull/170/commits/b3b325813d3aca5a5533763cc5b8f2e3a72076d2)